runtime/integration-tests/src/homa_lite_tests.rs

```diff
@@ -0,0 +1,174 @@
+// This file is part of Acala.
+
+// Copyright (C) 2020-2021 Acala Foundation.
+// SPDX-License-Identifier: GPL-3.0-or-later WITH Classpath-exception-2.0
+
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see <https://www.gnu.org/licenses/>.
+
+//! Tests the Homa-lite module, and its cross-chain functionalities.
+
+#[cfg(any(feature = "with-mandala-runtime", feature = "with-karura-runtime"))]
+use crate::integration_tests::*;
+#[cfg(feature = "with-karura-runtime")]
+use crate::kusama_test_net::*;
+
+use frame_support::{assert_noop, assert_ok};
+use orml_traits::MultiCurrency;
+
+#[cfg(feature = "with-karura-runtime")]
```

Comment:
code with different cfg should be split into different sub mod, instead repeating these cfg all the time